width: 1280           #图像宽度
height: 720            #图像高度
FrameRateEnable: true 
FrameRate: 100          #帧率
ExposureTime: 10000      #曝光时间,zlx的源码是1000的
GammaEnable: true
Gamma: 0.45              #伽马因子
Gain: 5                 #亮度
SaturationEnable: true
Saturation: 128         #饱和度
